enhancements/network/router-ca-configmap.md

```diff
@@ -0,0 +1,201 @@
+---
+title: router-ca-configmap
+authors:
+  - "@Miciah"
+reviewers:
+  - "@danehans"
+  - "@deads2k"
+  - "@frobware"
+  - "@ironcladlou"
+  - "@knobunc"
+approvers:
+  - "@deads2k"
+  - "@ironcladlou"
+  - "@knobunc"
+creation-date: 2019-11-20
+last-updated: 2019-11-20
+status: implemented
+see-also:
+replaces:
+superseded-by:
+---
+
+# `router-ca` configmap
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [X] Design details are appropriately documented from clear requirements
+- [X] Test plan is defined
+- [X] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+- Do consumers of `router-ca` care about non-default IngressControllers?
+- Should the ingress operator publish `router-ca` if a non-default
+  IngressController has an operator-generated certificate?
+- Should the operator publish `router-ca` even if the default IngressController
+  has a custom default certificate?
+- If so, should the operator publish the default certificate itself to
+  `router-ca`?
+- If the operator publishes custom default certificates to `router-ca`,
+  and if more than one IngressController exists, should the operator publish
+  a bundle with all custom default certificates for all IngressControllers,
+  just the certificate for the default IngressController (if it exists),
+  or just for whatever IngressController serves the ingress domain (if any)?
+- If a custom default certificate is signed by a certificate configured
+  as the trusted CA in the cluster proxy configuration, should the operator
+  publish that certificate to `router-ca`?
+- Should we define a formal API in the cluster configuration?
+  (See "Alternatives" below.)
+
+## Summary
+
+The ingress operator publishes a `router-ca` configmap in the
```

Comment:
This is what it does now, correct?  Not a summary of what will be added?

Reply:
Correct.  This describes the current behavior.